dts/bindings/gpio/hpmicro,hpm-gpio.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 hpmicro
+# SPDX-License-Identifier: Apache-2.0
+
+description: HPMicro GPIO node
+
+compatible: "hpmicro,hpm-gpio"
+
+include: [gpio-controller.yaml, base.yaml, pinctrl-device.yaml]
+
+properties:
+    reg:
+      required: true
+
+    hpmicro-gpio-port:
```

Comment:
remove this, `hpmicro-gpio-port` seems to only ever be 1.

Reply:
The io port of the hpm6750evkmini development board is (1)gpiob. As a driver, all possible io configurations should be reserved.